src/components/MultiHopTrade/components/PriceImpact.tsx

```diff
@@ -0,0 +1,36 @@
+import { Text as CText, useColorModeValue } from '@chakra-ui/react'
+import type { FC } from 'react'
+import { MdOfflineBolt } from 'react-icons/md'
+import { useTranslate } from 'react-polyglot'
+import { HelperTooltip } from 'components/HelperTooltip/HelperTooltip'
+import { Row } from 'components/Row/Row'
+import { RawText, Text } from 'components/Text'
+
+interface PriceImpactProps {
+  impactPercentage: string
+}
+
+export const PriceImpact: FC<PriceImpactProps> = ({ impactPercentage }) => {
+  const translate = useTranslate()
+  const redText = useColorModeValue('red.500', 'red.400')
+
+  return (
+    <Row fontSize='sm' flex={1}>
+      <Row.Label display='flex' alignItems='center' gap={1}>
+        <Text translation='trade.priceImpact' />
+        <RawText as='span' fontSize='lg' color='gray.500'>
+          <MdOfflineBolt />
+        </RawText>
+      </Row.Label>
+      <Row.Value>
+        <HelperTooltip
+          label={translate('trade.tooltip.priceImpact')}
+          flexProps={{ flexDirection: 'row' }}
+          iconProps={{ color: redText }}
```

Comment:
Extract me (props object) for render performance improvements.

Reply:
https://github.com/shapeshift/web/pull/5033/commits/40e7adaa2388993eff19afbb4887fbc26644e98c